components/producers/checkov/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"flag"
+	"log"
+
+	"github.com/go-errors/errors"
+
+	v1 "github.com/ocurity/dracon/api/proto/v1"
+	"github.com/ocurity/dracon/pkg/cyclonedx"
+	"github.com/ocurity/dracon/pkg/sarif"
+
+	"github.com/ocurity/dracon/components/producers"
+)
+
+// the CycloneDX target override
+var Target string
```

Comment:
why does the Target need to be a public variable? also, this pattern is very easy to cause issues and weird interactions between tests, it should be passed as a parameter in a simpler manner not as a global variable